src/main/msp/msp_serial.c

```diff
@@ -120,40 +196,84 @@ static uint8_t mspSerialChecksumBuf(uint8_t checksum, const uint8_t *data, int l
     return checksum;
 }
 
+static uint8_t mspSerialChecksumBufV2(uint8_t checksum, const uint8_t *data, int len)
```

Comment:
This is a generic dvb_s2 checksum and so should be in `maths.c` next to `crc8_dvb_s2`.